src/db/clients/mysql.js

```diff
@@ -143,17 +144,18 @@ export function listTableTriggers(client, table) {
 }
 
 export function executeQuery(client, query) {
+  const commands = identifyCommands(query);
   return new Promise((resolve, reject) => {
     client.query(query, (err, data, fields) => {
       if (err && err.code === mysqlErrors.ER_EMPTY_QUERY) return resolve([]);
       if (err) return reject(_getRealError(client, err));
 
       if (!isMultipleQuery(fields)) {
```

Comment:
I guess not. I will try to test it out of this commit and check if the tests pass.